RELEASE.md

```diff
@@ -3,6 +3,8 @@
 ## Major features and improvements
 * `pipeline` now accepts `tags` and a collection of `Node`s and/or `Pipeline`s rather than just a single `Pipeline` object. `pipeline` should be used in preference to `Pipeline` when creating a Kedro pipeline.
 * `pandas.SQLTableDataSet` and `pandas.SQLQueryDataSet` now only open one connection per database, at instantiation time (therefore at catalog creation time), rather than one per load/save operation.
+* Added new command group, `micropkg`, to replace `kedro pipeline pull` and `kedro pipeline package` with `kedro micropkg pull` and `kedro micropkg package` for Kedro 0.18.0.
+* Added support for bulk packaging/pulling micro-packages using `kedro pipeline package/pull --all` and `pyproject.toml`.
```

Comment:
This last one doesn't look right. Functionality was already there. I meant the migration notes should mention [tool.blabla.micropkg] instead of [tool.blabla.pipeline], but I guess that's for `develop` PR.